engine/installation/linux/docker-ee/suse.md

```diff
@@ -66,14 +68,35 @@ networks, are preserved. The Docker EE package is now called `docker-ee`.
 ## Configure the btrfs filesystem
```

Comment:
BTRFS

Reply:
Why? https://btrfs.wiki.kernel.org/index.php/Main_Page calls it `Btrfs` in the page and `btrfs` in the page title. I'm pretty OK with leaving it lowercase.